src/cdk-new/create-access-log-group.ts

```diff
@@ -0,0 +1,19 @@
+import type {ILogGroup} from 'aws-cdk-lib/aws-logs';
+import {LogGroup, RetentionDays} from 'aws-cdk-lib/aws-logs';
+import type {Stack} from 'aws-cdk-lib/core';
+import {RemovalPolicy} from 'aws-cdk-lib/core';
+
+export interface AccessLogGroupInit {
+  readonly stack: Stack;
+  readonly domainName: string;
+}
+
+export function createAccessLogGroup(init: AccessLogGroupInit): ILogGroup {
+  const {stack, domainName} = init;
+
+  return new LogGroup(stack, `AccessLogGroup`, {
+    logGroupName: `/aws/apigateway/accessLogs/${domainName}}`,
```

Comment:
Is this really a domain name? or something like: `aws--simple--awesome-app-v0-33-1` ?
Ideally, you could influence the name from the config. We're currently using something like `${appName}-${appVersion}`

Reply:
Yes, according to Helge, this is the standard naming for such kind of log groups. In the future, the concept of app names and versions will no longer exist at aws-simple. The combination of HostedZoneName and possibly subdomain name is unique. Both combined correspond to the domain name used here.